hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/OMDBInsightEndpoint.java

```diff
@@ -139,6 +161,9 @@ public Response getOpenKeyInfo(
     KeyInsightInfoResponse openKeyInsightInfo = new KeyInsightInfoResponse();
     List<KeyEntityInfo> nonFSOKeyInfoList =
         openKeyInsightInfo.getNonFSOKeyInfoList();
+
+    // Create a HashMap for the clusterSummary
+    Map<String, Object> clusterSummary = new HashMap<>();
```

Comment:
I think this should not be named as "clusterSummary", can we rename as "keysSummary"

Reply:
Done!